src/app/footer/footer.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
```

Comment:
seems like that file can be replaced with inline html as it is really small

Reply:
Actually now it is big enought